sys/const.go

```diff
@@ -65,186 +65,6 @@ var (
 
 	FaucetAddress = "0f569c84d434fb0ca682c733176f7c0c2d853fce04d95ae131d2f9b4124d93d8"
 
-	GasTable = map[string]uint64{
```

Comment:
Can we keep this with a `nolint` and open an issue to make it used ?

Reply:
ok